gaming-website/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:16
```

Comment:
Any particular reason such an old version of node is used?

Reply:
No particular reason, we can use the latest version that i just tested :)

Reply:
Actually is better having the last version rather then relying on an older one even is working for the time being